commands/login.js

```diff
@@ -22,63 +16,20 @@ module.exports = new Command('login')
       return utils.reject('Cannot run login in non-interactive mode. Pass ' + chalk.bold('--token') + ' instead.', {exit: 1});
```

Comment:
I think we need to update this message, maybe:

Cannot run login in non-interactive mode. See login:ci to generate a token for use in non-interactive environments.

Don't love that wording, but think the current is a bit confusing.